services/static-webserver/client/source/class/osparc/utils/LibVersions.js

```diff
@@ -60,6 +60,18 @@ qx.Class.define("osparc.utils.LibVersions", {
       return url;
     },
 
+    getVcsReleaseDate: function() {
```

Comment:
cool. Those are coming in https://github.com/ITISFoundation/osparc-simcore/pull/3861

Reply:
👍